Sprint-2/implement/contains.test.js

```diff
@@ -33,3 +45,11 @@ test.todo("contains on empty object returns false");
 // Given invalid parameters like an array
 // When passed to contains
 // Then it should return false or throw an error
+test("input is not an object", () => {
+  expect(contains(["a", "q", 5], "a")).toBe(false);
+  expect(contains("hello", "e")).toBe(false);
+  expect(contains(["a", "q", 5], 5)).toBe(false);
```

Comment:
Try checking also 
`expect(contains(["a", "q", 2], 2)).toBe(false);`

Reply:
with amending my function from last comment, this test is working as well.
Thanks